### PR TITLE
Update data-loader to utilize Key properties in the model

### PR DESCRIPTION
**Pull request by wfy1997**

### CODE REVIEW

**Review by n2iw — CHANGES_REQUESTED** [review]
When using ICDC model, I got following errors:
`Traceback (most recent call last):
  File "/Users/yingm3/work/code/icdc/data-loader/./loader.py", line 318, in <module>
    main(parse_arguments())
  File "/Users/yingm3/work/code/icdc/data-loader/./loader.py", line 230, in main
    schema = ICDC_Schema(config.schema_files, props)
  File "/Users/yingm3/work/code/icdc/data-loader/icdc_schema.py", line 99, in __init__
    node_id_list = self.get_node_id(node_type)
  File "/Users/yingm3/work/code/icdc/data-loader/icdc_schema.py", line 111, in get_node_id
    for prop in self.org_schema[NODES][node_type][PROPERTIES]:
TypeError: 'NoneType' object is not iterable`

**Review by n2iw — CHANGES_REQUESTED** [review]

File: icdc_schema.py [hunk 1]
```diff
@@ -93,6 +91,29 @@ def __init__(self, yaml_files, props):
...
+                elif len(node_id_list) > 1:
+                    raise Exception("More than one key property found for the same node")
+        if len(id_fields) > 0:
+            self.props.id_fields = id_fields
+            
+
+    def get_node_id(self, node_type):
+        node_id_list = []
+        for prop in self.org_schema[NODES][node_type][PROPERTIES]:
+            if KEY in self.org_schema[PROP_DEFINITIONS][prop]:
```
[the remark this answers is not in the document]
Comment (wfy1997) [reply icdc_schema.py:112]: It will handle the Key: false situation
with code "if self.org_schema[PROP_DEFINITIONS][prop][KEY]:"

**Review by n2iw — CHANGES_REQUESTED** [review]
It seems when creating indexes, the Key properties read from model file are not included. You can test this by creating a new database, then remove some ID fields from prop file, then observe the output related to index creation.

**Review by n2iw — APPROVED** [review]
LGTM

**Pull request merged**